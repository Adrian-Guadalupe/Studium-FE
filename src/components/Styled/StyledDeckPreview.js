import React from 'react';
import styled from 'styled-components';
import PropTypes from 'prop-types';
import {Card} from 'antd';

const StyledDeckPreview = ({

})

export default StyledDeckPreview;