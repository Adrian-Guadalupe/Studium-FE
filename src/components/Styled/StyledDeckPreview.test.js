import React from 'react';
import renderer from 'react-test-renderer';

import StyledDeckPreview from './StyledDeckPreview';

test('StyledDeckPreview render correctly', () => {
    const tree = renderer.create(<StyledDeckPreview />).toJSON();
    expect(tree).toMatchSnapshot();
});