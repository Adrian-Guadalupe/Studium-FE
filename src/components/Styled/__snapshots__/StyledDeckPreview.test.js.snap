// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledDeckPreview render correctly 1`] = `
.c0.c0 {
  width: 97px;
  height: 153px;
  margin-top: 20px;
  border-radius: 13px;
  border: 3px solid rgba(136,136,136,0.75);
  box-sizing: border-box;
}

<div
  className="ant-card c0 ant-card-bordered ant-card-type-inner"
>
  <div
    className="ant-card-body"
    style={Object {}}
  />
</div>
`;
